api.go

```diff
@@ -492,8 +492,8 @@ func (d *Context) GetLength(index int) int {
 }
 
 // See: http://duktape.org/api.html#duk_get_lstring
-func (d *Context) GetLstring(index int, outLen int) string {
-	if s := C.duk_get_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
+func (d *Context) GetLstring(index int, outLen *int) string {
```

Comment:
Without pointer you can't get the buffer outLen outside of a GetLstring function. E.g. in the example that I provided I need to get the length of a buffet with bytecode. I'm not sure my English is good enough to explain whay I mean so I provided link to code earlier.